core-common/src/main/java/org/apache/kylin/common/util/EncryptUtil.java

```diff
@@ -32,6 +32,9 @@
             0x65, 0x79 };
 
     public static String encrypt(String strToEncrypt) {
+        if (strToEncrypt == null) {
```

Comment:
it's better use 'org.apache.commons.lang3.StringUtils.isBlank' to check blank string case.

Reply:
Hi, zzcclp:
Thanks for the suggestion. But actually, a blank string can be encrypted and its cypher text is not a blank string. 
e.g. result of EncryptUtil.encrypt("") is "719bTBvphWewRiCiCj3D/A=="
So only null check is needed, I think.

Reply:
Got it. Thanks for your explanation.